### PR TITLE
Generating sysinfoapi and winnt bindings

### PR DESCRIPTION
**Pull request by nathan-alden-sr**
I also included the changes we discussed due to the new version of ClangSharp.

### CODE REVIEW

File: generation/um/sysinfoapi/generate.rsp [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+_LOGICAL_PROCESSOR_RELATIONSHIP=LOGICAL_PROCESSOR_RELATIONSHIP
+_MEMORYSTATUSEX=MEMORYSTATUSEX
+_OSVERSIONINFOA=OSVERSIONINFOA
+_OSVERSIONINFOW=OSVERSIONINFOW
+_SYSTEM_INFO=SYSTEM_INFO
+_SYSTEM_LOGICAL_PROCESSOR_INFORMATION=SYSTEM_LOGICAL_PROCESSOR_INFORMATION
+_SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX=SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX
+_SYSTEM_PROCESSOR_CYCLE_TIME_INFORMATION=SYSTEM_PROCESSOR_CYCLE_TIME_INFORMATION
+_SYSTEMTIME=SYSTEMTIME
+DWORD_PTR=uint*
```
Comment (tannergooding) [inline generation/um/sysinfoapi/generate.rsp:37]: Was this one not handled automatically?

Comment (nathan-alden-sr) [reply generation/um/sysinfoapi/generate.rsp:37]: Correct. I just generated bindings again without `DWORD_PTR` and it generated `uint`.

Comment (tannergooding) [reply generation/um/sysinfoapi/generate.rsp:37]: Looks like that's because it is `typedef ULONG_PTR DWORD_PTR` and so therefore should be `UIntPtr`

Comment (nathan-alden-sr) [reply generation/um/sysinfoapi/generate.rsp]: As discussed, I changed the mapping to `DWORD_PTR=UIntPtr`.

**Pull request merged**